PresentScreenings/Controllers/ViewController.cs

```diff
@@ -539,10 +546,17 @@ public static void ReportCoincidingScreeninings()
                 .Select(pair => $"{line}{pair.Key}:{line}{string.Join(',' + line, pair.Value.Select(s => s.ScreeningTitle))}");
             if (coinciders.Count() > 0)
             {
-                string title = "Coinciding Screenings Found";
+                // Compose coinsiders info.
                 StringBuilder builder = new StringBuilder($"Coinciding screenings:{line}");
                 builder.AppendJoin(line, coinciders);
-                AlertRaiser.RunInformationalAlert(title, builder.ToString(), true);
+
+                // Write coinsiders info.
+                AlertRaiser.WriteWarning(builder.ToString());
+
+                // Display where the files have been stored.
+                string title = "Coinciding Screenings Found";
+                string text = $"Coinsiders report is saved in {AlertRaiser.WarningFile}";
+                AlertRaiser.RaiseNotification(title, text);
```

Comment:
Please consider to replace all alerts by notifications (or get alerts working in Ventura).

Reply:
Alerts do work under Ventura, but seemingly not when no window is finished launching.

Added code to `QuitWithAlert()` that raises a notification and writes to the error file when the alert doesn't return 0.